src/style.css

```diff
@@ -1 +1,102 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+body {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+}
+
+.search_bar {
+  height: 68px;
+  width: 100%;
+  margin-top: 20px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Here you've added a top margin, which could lead to margin collapse if there are elements above with a bottom margin.